src/elements/copy.js

```diff
@@ -1,5 +1,12 @@
 export default function(namespace) {
   const StringElement = namespace.getElementClass('string');
+  const ArrayElement = namespace.getElementClass('array');
+
+  ArrayElement.prototype.__defineGetter__('copy', function() {
+    return this.children((item) => item.element === 'copy');
+  });
+
+  namespace.register('array', ArrayElement);
```

Comment:
Is this register necessary? Aren't you modifying the prototype of the already-registered class in-place?